lib/ohai/plugins/linux/network.rb

```diff
@@ -276,7 +276,7 @@ def link_statistics(iface, net_counters)
         end
       end
 
-      if line =~ /state (\w+)/
+      if line =~ /\sstate (\w+)/
```

Comment:
Can you add a comment above with a permalink from https://rubular.com/ that contains the output you trying to match. This is really helpful for us years later when we try to figure out what these regexes are trying to match.

Reply:
I have these two lines:
```
... noqueue state UP mode DEFAULT group default qlen 1000 ...
... bridge forward_delay 1500 hello_time 200 max_age 2000 ageing_time 30000 stp_state 0 priority 32768 ...
```
The network plugins thinks my interface is down because of `stp_state 0`.